pkg/aws/eni/node_manager_test.go

```diff
@@ -127,8 +127,7 @@ func (e *ENISuite) TestNodeManagerDefaultAllocation(c *check.C) {
 	c.Assert(mngr, check.Not(check.IsNil))
 
 	// Announce node wait for IPs to become available
-	cn := newCiliumNode("node1", withInstanceID(instanceID), withInstanceType("m5.large"),
-		withAvailabilityZone("us-west-1"), withVpcID("vpc-1"), withIPAMPreAllocate(8))
+	cn := newCiliumNode("node1", withInstanceID(instanceID), withInstanceType("m5.large"), withIPAMPreAllocate(8))
```

Comment:
nit: I feel a bit torn about this last commit, since it increases the coupling of `newCiliumNode` (which was value environment agnostic before this change) with the mock resources. 

Have you considered adding a `withTestDefaults()` helper (which sets the test vpc and az) instead?

Reply:
Ah very true, implemented as suggested.